### PR TITLE
Address Woo Quality Insight recommendations

### PR DESCRIPTION
**Pull request by mntzrr**
https://app.clickup.com/t/8694w2y1z

### CODE REVIEW

File: classes/class-kco-credentials.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public function get_credentials_from_session() {
 
 		$credentials = array(
 			'merchant_id'   => $this->settings[ $test_string . 'merchant_id_' . $country_string ],
-			'shared_secret' => htmlspecialchars_decode( $this->settings[ $test_string . 'shared_secret_' . $country_string ] ),
+			'shared_secret' => htmlspecialchars_decode( $this->settings[ $test_string . 'shared_secret_' . $country_string ], ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401 ),
```
Comment (MichaelBengtsson) [inline classes/class-kco-credentials.php:58]: Why is the `ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401` part needed? This is the default for the [PHP function](https://www.php.net/manual/en/function.htmlspecialchars-decode.php)

File: includes/kco-functions.php [hunk 2]
```diff
@@ -97,6 +97,7 @@ function kco_wc_show_snippet( $pay_for_order = false ) {
 
 	if ( isset( $klarna_order['html_snippet'] ) ) {
 		do_action( 'kco_wc_show_snippet', $klarna_order );
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- We trust the HTML snippet.
```
Comment (MichaelBengtsson) [inline includes/kco-functions.php:100]: Should this not be on the same line as the echo for it to work properly?

**Review by MichaelBengtsson — CHANGES_REQUESTED** [review]

File: classes/class-kco-credentials.php (hunk 1 above)
Comment (mntzrr) [reply classes/class-kco-credentials.php:58]: image

File: includes/kco-functions.php (hunk 2 above)
Comment (mntzrr) [reply includes/kco-functions.php:100]: It will ignore the current and next line: https://stackoverflow.com/a/59073074

**Review by MichaelBengtsson — APPROVED** [review]

**Pull request merged**